tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Storages.resource

```diff
@@ -119,3 +119,11 @@ Get Openshift PVC From Storage
     [Arguments]     ${name}      ${namespace}
     ${rc}  ${pvc_name}=    Run And Return Rc And Output   oc get pvc -n ${namespace} -o jsonpath='{.items[?(@.metadata.annotations.openshift\\.io/display-name=="${name}")].metadata.name}'
     RETURN    ${rc}    ${pvc_name}
+
+Delete PVC From CLI
```

Comment:
In PVCChange.resource there is already the keyword `May Be Delete PVC`. 

I understand this keyword is specific for Data Science Projects. What about just calling it `Delete Storage From CLI`?

Reply:
Yeah, this kw is specific to DSProject because it needs fetch more data. If you want I can try to merge the 2 kw. 

> What about just calling it Delete Storage From CLI

yeah, I'm trying to stick with "PVC" because, based on UX prototypes, there might be different type of storages. So, in order to avoid very big changes in the code in future, I chose to make kw referring to PVC explicitly.